brainstormforce/ultimate-addons-for-gutenberg
# Fixed: File Generation issue on Woocommerce Pages.

### Description
Fixed: File Generation issue on Woocommerce Pages.

### Types of changes
 Bugfix (non-breaking change which fixes an issue) 

### How has this been tested?
 - Enable File Generation from **Settings > UAG.**
 - Check if CSS applied on Woocommerce Pages.

### Checklist:
